src/events/guildMemberRemove.ts

```diff
@@ -76,5 +77,12 @@ export function makeHandler({ database, challonge }: CommandSupport) {
 				}
 			})
 			.catch(logger.error);
+		const manualParticipants = await ManualParticipant.find({
+			where: { discordId: member.id },
+			relations: ["tournament"]
+		});
+		for (const participant of manualParticipants) {
+			await dropPlayer(participant.tournament, participant, member);
+		}
```

Comment:
Uh, this is not what the original query was. Now the participant is getting removed from every tournament everywhere as soon as they leave any server.